docs/getting-started/getting-started-with-docker.rst

```diff
@@ -122,13 +125,13 @@ This switches to the ``root`` user to perform software installs. It
 downloads BAMStats, unzips it, and installs it in the correct location,
 ``/opt``.
 
-**This is why Docker is so powerful.** On HPC systems the above process
+This is one of the main advantages of Docker. On some systems the above process
 might take days or weeks of working with a sys admin to install
-dependencies on all compute nodes. Here I can control and install
-whatever I like inside my Docker image - correctly configuring the
-environment for my tool and avoiding the time to set up these
-dependencies in the places I want to run. This greatly simplifies the
-install process for other users that you share your tool with as well.
+dependencies. Instead of installing dependencies on every machine someone might want to run a workflow/tool on, we can instead control and install
```

Comment:
Suggestion: 

Instead of installing dependencies on every machine one might want to run a workflow/tool on, we can instead install whatever we like inside a Docker image. In this way, we can correctly configure the
environment for a tool and avoid the time to set up these
dependencies in all the systems we might want to run the workflow/tool on. This greatly simplifies the
install process for other users.

Reply:
I split one long sentence into two. And made some subtle changes.

Reply:
I tried to iterate on what you wrote a little bit for greater clarity. How about this?

> This is one of the main advantages of Docker. On some systems the above process might take days or weeks of working with a sysadmin to install dependencies. Instead of installing dependencies on every machine someone might want to run a workflow/tool on, we can create a Docker image that wraps that workflow/tool. With Docker, we only need to correctly configure the environment and dependencies for a program once, and once that Docker image is set up, we avoid repeating that process. All we need to do is transfer that Docker image to other machines, and they will be able to run what's inside it. This greatly simplifies the process for other users.